### ISSUE TITLE
We should update our Welcome modals

### ISSUE THREAD

**Issue by mazameli** [metabase/metabase#5139]
For context, here are the current ones:

![welcome-modal-1](https://cloud.githubusercontent.com/assets/2223916/26217200/6b8df518-3bbb-11e7-9cf1-0b03f38ee7d2.png)
![welcome-modal-2](https://cloud.githubusercontent.com/assets/2223916/26217199/6b7a998c-3bbb-11e7-874d-6d6c35048e2d.png)
![welcome-modal-3](https://cloud.githubusercontent.com/assets/2223916/26217201/6b9c6576-3bbb-11e7-90db-464b1d897603.png)

I think they're currently too wordy, outdated, and don't do a great job of highlighting some of Metabase's best features (particularly drill-through). I also think we should no longer be driving people to the query builder tutorial.

These aren't necessarily final designs, but what I was trying to do was group some of our best features together into a set of 3 highly visual modals to give first time users a better overview of what Metabase can do. I liked leading with drill-through, particularly when considering users who come on board after user #1, though maybe it's a little funny for that very first user, considering they won't have any existing dashboards/questions — though this gets a lot more logical if we do something like add an example dashboard (or auto-magic dashboards!) to new instances.

![new-welcome-1](https://cloud.githubusercontent.com/assets/2223916/26217940/1a5bce60-3bbe-11e7-9c0f-1ac445d8b401.png)

![new-welcome-2](https://cloud.githubusercontent.com/assets/2223916/26217943/1e424068-3bbe-11e7-90fc-6cbe1a07fd80.png)

![new-welcome-3 v2](https://cloud.githubusercontent.com/assets/2223916/26217949/2251d7a4-3bbe-11e7-8654-0d03e3e9e909.png)

**Some alternative approaches I considered but didn't execute:**
- We could ditch the modals entirely and instead include these three bits of content or something like them in the Activity Feed. This would make the home page feel less blank, and would also allow the content to persist and possibly act as jumping-off points. The con is that I'm not sure how much we want to double-down on the Activity Feed if we think we might move it or otherwise change the home page in a big way.
- Instead of these modals, we could show either modals or more detailed empty states on the various sections of the product. This isn't truly an alternative, since it's not a mutually exclusive path, and we're already sort of doing this (i.e., we now have better empty states for the Questions and Dashboards pages that feel more like the Pulses empty state)

**Comment by salsakran** [comment]
#ShipIt

**Comment by kdoh** [comment]
Done in #5143
